neon/data-collection/scene-camera-exposure/index.md

```diff
@@ -0,0 +1,44 @@
+# Scene Camera Exposure
+The [scene camera’s](https://docs.pupil-labs.com/neon/data-collection/data-streams/#scene-video) exposure modes can be adjusted to improve image quality in different lighting conditions. There are four modes:
+
+- **Manual:** This mode lets you set the exposure value manually.
+- **Automatic**: `Highlights`, `Balanced`, and `Shadows` modes automatically adjust exposure according to the surrounding lighting.
+    - `Highlights` mode optimizes the exposure to capture bright areas in the environment, while potentially underexposing dark areas.
+    - `Balanced` mode optimizes the exposure to capture brighter and darker areas in the environment equally.
+    - `Shadows` mode optimizes the exposure to capture darker areas in the environment, while potentially overexposing bright areas.
+
+::: tip 
+The mode you choose should depend on the environment. The images below provide an example and some caveats.
+:::
+
+## Changing Exposure Modes
+
+From the home screen of the Neon Companion app, tap the [Scene and Eye Camera preview](https://docs.pupil-labs.com/neon/data-collection/first-recording/#_4-open-the-live-preview). ![Scene Exposure UI](UI_small.webp)
```

Comment:
I think this image is probably unnecessary. But keen to hear what others think.

Reply:
I think the image only helps if we can do 2 columns (side-by-side text + image). Since we can't do it, the section looks a bit awkward.